### PR TITLE
docs: add WebStorm plugin support for Vitest

### PR DESCRIPTION
**Pull request by linux-china**
Add Vitest WebStorm plugin

### CODE REVIEW

File: docs/guide/ide.md [hunk 1]
```diff
@@ -6,8 +6,20 @@
...
 </p>
 
-We are now providing a official extension for Visual Studio Code to enhance your testing experience with Vitest.
+We are now providing an official extension for Visual Studio Code to enhance your testing experience with Vitest.
 
 [GitHub](https://github.com/vitest-dev/vscode) | [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer)
 
 ![](https://i.ibb.co/bJCbCf2/202203292020.gif)
+
+## WebStorm
```
Comment (userquin) [inline docs/guide/ide.md:15]: add also `IntelliJ` => `IntelliJ/WebStorm`

Comment (userquin) [reply docs/guide/ide.md:15]: also the image?

**Comment by userquin** [comment]
uhmm, these urls should also be included on the sw manifest (maybe we can donwload them to local, I need to review the docs)

@linux-china you don't need to do it, it is just a reminder

**Comment by userquin** [comment]
@linux-china can you make the `+` separator bigger (between Vitest and JetBrains logos)?

**Comment by linux-china** [comment]
> @linux-china can you make the `+` separator bigger (between Vitest and JetBrains logos)?
> 
> EDIT: I mean, it is really small, try to match the size on the VSCode image...

Yes, it's too small and I adjusted it already.

**Review by userquin — APPROVED** [review]

**Comment by antfu** [comment]
Awesome! Thank you.

A bit nit-picking tho, the logo does not seem to render well in dark mode:
image
It would be great if you could take a bit more time to make the shadow transparent :)

**Pull request merged**